scripts/kind-e2e/lib_helm_deploy_subm.sh

```diff
@@ -56,9 +56,17 @@ function setup_broker() {
 
 function helm_install_subm() {
     cluster_id=$1
-    cluster_cidr=$2
-    service_cidr=$3
-    crd_create=$4
+    crd_create=$2
+    cluster_cidr=$3
+    service_cidr=$4
+    global_cidr=$5
+    if [ -z $global_cidr ]
```

Comment:
can be simplified as:

`globalnet_enable="true" && [[ -z $global_cidr ]] && globalnet_enable="false"`

Reply:
Done